Feature Add automatic seed file creation

BenSeDev opened the pull request:
This PR will allow you to create a seed file in the same way as you create a migration file.

Similar to creating a new migration file:
`vendor/bin/migraine new db_name "description which becomes the file name"`

With this update you'll be able to run the following:
`vendor/bin/migraine new-seed db_name "description which becomes the file name"`

It will then create the seed file inside the appropriate seeds folder inside the folder named after the DB.

BenSeDev committed:
♻️ Exclude phpstorm settings from vcs

BenSeDev committed:
✨ Add symfony/filesystem to required packages

This way we don't have to include this in the Dockerfile

BenSeDev committed:
💅 Update docblock

BenSeDev committed:
✨ Make it possible to process a seed command

BenSeDev committed:
✨ Add Seed command

BenSeDev committed:
✨ Add Seed command to application for usage

BenSeDev committed:
✨ Remove version

this was only needed for local testing...

cedricengagor reviewed (commented):
Wouldn't it make sense if we took the approach:
```sh
migraine new migration test "create user table"
migraine new seed test "default user"
```

This would basically follow the pattern:
`migraine {action=new} {type=seed} {group=test} {name="default user"}`

BenSeDev committed:
✨ Add migraine type to the CLI

When creating a new file, the first argument after new should be the type of file you want to create (migration or seed)

BenSeDev committed:
✨ Rename type to action

BenSeDev committed:
✨ Rename type to action

Commit:
Update src/CommandNew.php

Co-authored-by: Matthias De Schoenmacker <75781401+matthias-deschoenmacker@users.noreply.github.com>

BenSeDev committed:
✨ Update README.md & CHANGELOG.md with breaking changes

BenSeDev committed:
Merge remote-tracking branch 'origin/add-automatic-seed-creation' into add-automatic-seed-creation

BenSeDev committed:
✨ Update README.md on how to create a seed file

BenSeDev committed:
♻️ Fix typo

we want `new seed` and not `new migration`

Commit:
Update src/NewSeed.php

Co-authored-by: cedricengagor <32090836+cedricengagor@users.noreply.github.com>

BenSeDev committed:
✨ Add optional `--seed` param

Default no seeds will be executed, but if you want to, now in addition to all migrations, the seeds inside the seeds folder will be executed it the option --seed is provided during the migrate command

BenSeDev committed:
✨ Add optional `--seed` command option

Adding this to the migrate command will seed all files inside the seeds folder.

BenSeDev committed:
Merge remote-tracking branch 'origin/add-automatic-seed-creation' into add-automatic-seed-creation

BenSeDev committed:
✨ Add updated info in changelog & readme

BenSeDev committed:
✨ Split seed and migration logic into separate files.

Commit:
Update src/CommandMigrate.php

Co-authored-by: cedricengagor <32090836+cedricengagor@users.noreply.github.com>

Commit:
Update src/CommandNew.php

Co-authored-by: cedricengagor <32090836+cedricengagor@users.noreply.github.com>

Commit:
Update src/CommandNew.php

Co-authored-by: cedricengagor <32090836+cedricengagor@users.noreply.github.com>

BenSeDev committed:
👌 Add unit tests for SeedUp

BenSeDev committed:
Merge remote-tracking branch 'origin/add-automatic-seed-creation' into add-automatic-seed-creation

BenSeDev committed:
👌 Rename variables to seed instead of migration

BenSeDev committed:
👌 Extract query to variable

BenSeDev committed:
💅 Specify which row to select

BenSeDev committed:
🔥 Remove unused event

BenSeDev committed:
💅 Prepare for packagist

BenSeDev committed:
💅 Prepare for packagist

BenSeDev committed:
💅 Prepare for packagist
